configuration-examples/slotting-facet-options/README.md

```diff
@@ -0,0 +1,52 @@
+# Slotting Facet Options
+
+In this example, we explore the need to programatically slot facet options to achieve the desired order in Constructor's Search/Browse API Responses.
+
+## Considerations - Do I need to do this?
+
+Constructor's algorithms already arranges facet options dynamically, according to the personalized intent of each customer. The Customer Dashboard also allows you to change the arrangement of facet options according to ["Number of matches" or "Alpha-numerically"](https://docs.constructor.com/docs/using-the-constructor-dashboard-configuring-rules-facets-configure-global-facets#range-1), and gives you the abiliy to slot individual facets if needed.
+
+Given these options, we do not recommend programatically slotting your facet options unless:
+
+1. The facet you're looking to sort on has a typical order users would expect it to be in (i.e. size) and,
+2. The values for these facet options cannot be modified to fit the "Alpha-numeric" sort provided.
+
+## How to use
+
+1. Step 1: Prepare a .csv file that contains the list of facet options as ingested in your catalog feed, and the corresponding position to slot it. The csv file should have two columns:
+   1. Column 1: The Facet Option to be slotted, as ingested in the catalog feed
```

Comment:
Should we be specific here and say `The Facet Option value (id)`?

Reply:
I think we can add the `value` but omit the `id` since `id` as a term doesn't really get used in the facet configuration endpoints